lib/linter-rust.coffee

```diff
@@ -25,19 +25,23 @@ class LinterRust
       exit = (code) ->
         return resolve [] unless code is 101 or code is 0
         messages = []
-        regex = XRegExp('(?<file>.+):(?<line>\\d+):(?<col>\\d+):\\s*(\\d+):(\\d+)\\s+((?<error>error|fatal error)|(?<warning>warning)|(?<info>note)):\\s+(?<message>.+)\n', '')
+        regex = XRegExp('(?<file>.+):(?<from_line>\\d+):(?<from_col>\\d+):\\s*(?<to_line>\\d+):(?<to_col>\\d+)\\s+((?<error>error|fatal error)|(?<warning>warning)|(?<info>note)):\\s+(?<message>.+)\n', '')
         XRegExp.forEach results.join(''), regex, (match) =>
           type = if match.error
             "Error"
           else if match.warning
             "Warning"
+
+          if match.from_col == match.to_col
```

Comment:
In JS, yes, but not in CS, [it does that for you](http://coffeescript.org/#try:foo%20%3D%20123%0Abar%20%3D%20%27123%27%0A%0Aif%20foo%20%3D%3D%20bar%0A%20%20alert%20%27gnah%27).